src/ChatContainer.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import ChatItemContainer from './ChatItemsContainer';
+import * as ReactDOM from 'react-dom';
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 auto;
+  padding-bottom: 20px;
+`;
+
+export default (props) => {
+  const lastItem = useRef();
+  const container = useRef();
+
+  const list = props.chatData.map((item, index) => {
+    return (
+      <ChatItemContainer
+        chatList={item.chatList}
+        user={item.user}
+        ownerUserId={props.ownerUserId}
+        ref={index === props.chatData.length - 1 ? lastItem : undefined}
+      />
+    );
+  });
+
+  useEffect(() => {
+    const node = ReactDOM.findDOMNode(container.current);
+    const calculatedHeight = node.clientHeight;
+
+    window.scrollTo({
+      top: calculatedHeight,
+      behavior: 'smooth',
+    });
+  });
```

Comment:
```suggestion
  useEffect(() => {
    container.current.scrollTo({
      top: clientHeight,
      behavior: 'smooth',
    });
  });
```

저도 `useRef()`를 사용해서 스크롤하는 부분 구현 했었는데요, 저렇게 바로 Ref에 `scrollTo`를 써줘도 가능합니다!
그리고 React 공식 Docs에서는 `findDOMNode()`를 사용하는 것은 컴포넌트 추상화를 위반하기 떄문에 권장하지 않고, 또 함수형 컴포넌트에서는 사용이 불가능하다고 합니다!

근데 `clientHeight`라는 상수가 존재했었다니,,, 저는 그냥 무식하게 1000픽셀 아래로 스크롤하게 했었는데,,, 중요한거 하나 배웁니다!

Reply:
height를 구할 방법을 찾아봐야할 것 같아요.